### PR TITLE
Use hash2 instead of deprecated hash

### PR DESCRIPTION
**Pull request by ghost**
---

- [x] `./mach build -d` does not report any errors
- [x] `./mach test-tidy` does not report any errors
- [ ] These changes fix #__ (github issue number if applicable).

- [ ] There are tests for these changes OR
- [ ] These changes do not require tests because _____

---
This change is [Reviewable](https://reviewable.io/reviews/servo/servo/16761)

### CODE REVIEW

File: PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -3,8 +3,8 @@
 
 ---
 <!-- Thank you for contributing to Servo! Please replace each `[ ]` by `[X]` when the step is complete, and replace `__` with appropriate data: -->
-- [ ] `./mach build -d` does not report any errors
-- [ ] `./mach test-tidy` does not report any errors
+- [X] `./mach build -d` does not report any errors
+- [X] `./mach test-tidy` does not report any errors
```
Comment (jdm) [inline PULL_REQUEST_TEMPLATE.md]: The template should not be pre-checked; you should be making those changes in the description of this pull request, instead.

**Comment by jdm** [comment]
This looks good! Please [squash](https://ariejan.net/2011/07/05/git-squash-your-latests-commits-into-one/) your commits into one, and then we'll merge this!

**Comment by jdm** [comment]
Unfortunately that appears to have multiplied the commits instead of squashing them ;) I recommend `git reset --hard 4e390e2630f244e77032ded404ca885c3e3ed8a7 && git cherry-pick 021f2ac`.

**Comment by Ksbugbee** [comment]
Should be fixed now. Thanks!

**Comment by jdm** [comment]
@Ksbugbee We're still back to two separate commits. If you run `git rebase -i 0539302^` and mark "Squash and fixed comment" as "fixup", you should get the result we're looking for.

**Comment by Ksbugbee** [comment]
Ah much better! Thanks! @jdm

**Comment by jdm** [comment]
@bors-servo: r+
Thanks!

**Pull request merged**